package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "prepublish": "node ./apidocs/describe-builtin-models.js",
     "pretest": "jshint . && bower install",
-    "test": "node ./test.e2e/test-server.js node_modules/karma/bin/karma start --single-run --browsers PhantomJS"
+    "test": "node ./test.e2e/test-server.js node node_modules/karma/bin/karma start --single-run --browsers PhantomJS"
```

Comment:
we have found out that the `test-server.js` actually receives the rest of the command (`node_modules/karma/bin/karma start --single-run --browsers PhantomJS`) as parameter then spawning a child_process to run it instead of running it from npm scripts directly